### PR TITLE
Improve game dashboard

### PR DESCRIPTION
**Pull request by AlexandreSi**
A few naming improvements suggested by @tristanbob

### CODE REVIEW

File: newIDE/app/src/GameDashboard/GameDetailsDialog.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ import { useResponsiveWindowWidth } from '../UI/Reponsive/ResponsiveWindowMeasur
 export type GamesDetailsTab =
   | 'details'
   | 'builds'
-  | 'feedbacks'
+  | 'feedback'
```
Comment (ClementPasteau) [inline newIDE/app/src/GameDashboard/GameDetailsDialog.js:52]: haha, I remember changing feedback into feedbacks, looks like it was a bad move!

**Review by 4ian — APPROVED** [review]

File: newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ const FeedbackCard = ({
       console.error(`Unable to update comment: `, error);
       showErrorBox({
         message:
-          i18n._(t`Unable to change resolved status of feedback.`) +
+          i18n._(t`Unable to change read status of feedback.`) +
```
Comment (D8H) [inline newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js:99]: "Unable to mark feedbacks as read."?

Comment (AlexandreSi) [reply newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js:99]: Good idea

Comment (AlexandreSi) [reply newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js:99]: But no in fact, it is used to mark a feedback as unread to so I prefer having it as is and not have 2 sentences to translate

**Review by D8H — APPROVED** [review]

File: newIDE/app/src/GameDashboard/GameDetailsDialog.js (hunk 1 above)
Comment (AlexandreSi) [reply newIDE/app/src/GameDashboard/GameDetailsDialog.js:52]: Well it's to have the same cardinality as for the tab 😬

File: newIDE/app/src/GameDashboard/GameDetailsDialog.js [hunk 3]
```diff
@@ -343,8 +343,8 @@ export const GameDetailsDialog = ({
           >
             <Tab label={<Trans>Details</Trans>} value="details" />
             <Tab label={<Trans>Builds</Trans>} value="builds" />
-            <Tab label={<Trans>Feedbacks</Trans>} value="feedbacks" />
-            <Tab label={<Trans>Analytics</Trans>} value="analytics" />
+            <Tab label={<Trans>Player Feedback</Trans>} value="feedback" />
+            <Tab label={<Trans>Statistics</Trans>} value="analytics" />
```
Comment (tristanbob) [inline newIDE/app/src/GameDashboard/GameDetailsDialog.js:347]: "Analytics" are what businesses and enterprises use.

"Statistics" are what games use (sports, video games).  This sounds much more "fun".  :)

**Pull request merged**